command/pipeline/validate.go

```diff
@@ -99,6 +99,49 @@ var CommandValidate = &cli.Command{
 			Value:   false,
 		},
 
+		// RuleData Flags
+		&cli.StringFlag{
+			EnvVars: []string{"VELA_BRANCH", "PIPELINE_BRANCH", "VELA_BUILD_BRANCH"},
+			Name:    "branch",
+			Aliases: []string{"b"},
+			Usage:   "provide the build branch for the pipeline",
+		},
+		&cli.StringFlag{
+			EnvVars: []string{"VELA_COMMENT", "PIPELINE_COMMENT", "VELA_BUILD_COMMENT"},
+			Name:    "comment",
+			Aliases: []string{"c"},
+			Usage:   "provide the build comment for the pipeline",
+		},
+		&cli.StringFlag{
+			EnvVars: []string{"VELA_EVENT", "PIPELINE_EVENT", "VELA_BUILD_EVENT"},
+			Name:    "event",
+			Aliases: []string{"e"},
+			Usage:   "provide the build event for the pipeline",
+		},
+		&cli.StringFlag{
+			EnvVars: []string{"VELA_STATUS", "PIPELINE_STATUS", "VELA_BUILD_STATUS"},
+			Name:    "status",
+			Usage:   "provide the expected build status for the local validation",
+			Value:   "success",
+		},
+		&cli.StringFlag{
+			EnvVars: []string{"VELA_TAG", "PIPELINE_TAG", "VELA_BUILD_TAG"},
+			Name:    "tag",
+			Aliases: []string{"t"},
```

Comment:
wouldn't this conflict with: https://github.com/go-vela/cli/blob/ce8b11612533892175c7fd968bb94d600ec6d36a/cmd/vela-cli/main.go#L92 ?